tests/components/wled/test_sensor.py

```diff
@@ -34,47 +35,47 @@ async def test_sensors(
 
     # Pre-create registry entries for disabled by default sensors
     registry.async_get_or_create(
-        SENSOR_DOMAIN,
+        Platform.SENSOR,
```

Comment:
This makes sense since the entity registry api requires an entity platform domain for this parameter.